README.md

```diff
@@ -7,6 +7,9 @@ and the [hyper HTTP library](https://github.com/hyperium/hyper).
 [![Documentation](https://docs.rs/hyper-rustls/badge.svg)](https://docs.rs/hyper-rustls/)
 
 # Release history
+- 0.24.0 (2023-04-01):
+  * Upgrade to rustls 0.21.0, tokio-rustls 0.24.0 and webpki-roots 0.23.0.
+  * Add `ConnectorBuilder::enable_all_versions()` helper.
```

Comment:
Does the increase in MSRV merit a mention here? It went from 1.56.1 to 1.57 in https://github.com/rustls/hyper-rustls/commit/90d23a2bb75ec1193757e2d4f9d76b1308b8d844

Reply:
I don't think so? It will be meaningless to the vast majority of the audience.